lib/v2/screens/transfer/send/send_enter_data/send_enter_data_screen.dart

```diff
@@ -79,88 +79,98 @@ class SendEnterDataScreen extends StatelessWidget {
             }
           },
           child: Scaffold(
-            appBar: AppBar(title: const Text("Back")),
+            appBar: AppBar(
+              title: Text("Send".i18n),
+              backgroundColor: Colors.transparent,
+              elevation: 0.0,
```

Comment:
i think the default elevation defined in the global theme data is 0.0

Reply:
But if the global theme changes? This one needs to be 0.0, I think - that's what all the examples said. I have not tried a different value though.

Reply:
this should not override the default value we set in the theme.